packages/ember-htmlbars/tests/helpers/each_test.js

```diff
@@ -1,617 +0,0 @@
-  runDestroy(view);
-  view = EmberView.create({
-    items: [
-      { id: 1 },
-      { id: 2 },
-      { id: 3 }
-    ],
-    template: compile('{{#each view.items key=\'@index\' as |item|}}{{item.id}}{{/each}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '123');
-});
-
-QUnit.test('can specify `@identity` to represent primitive items', function() {
-  runDestroy(view);
-  view = EmberView.create({
-    items: [1, 2, 3],
-    template: compile('{{#each view.items key=\'@identity\' as |item|}}{{item}}{{/each}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '123');
-
-  run(function() {
-    set(view, 'items', ['foo', 'bar', 'baz']);
-  });
-
-  equal(view.$().text(), 'foobarbaz');
-});
-
-QUnit.test('can specify `@identity` to represent mixed object and primitive items', function() {
-  runDestroy(view);
-  view = EmberView.create({
-    items: [1, { id: 2 }, 3],
-    template: compile('{{#each view.items key=\'@identity\' as |item|}}{{#if item.id}}{{item.id}}{{else}}{{item}}{{/if}}{{/each}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '123');
-
-  run(function() {
-    set(view, 'items', ['foo', { id: 'bar' }, 'baz']);
-  });
-
-  equal(view.$().text(), 'foobarbaz');
-});
-
-QUnit.test('duplicate keys work properly with primitive items', function() {
-  runDestroy(view);
-  view = EmberView.create({
-    items: ['a', 'a', 'a'],
-    template: compile('{{#each view.items as |item|}}{{item}}{{/each}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'aaa');
-});
-
-QUnit.test('pushing a new duplicate key will render properly with primitive items', function() {
-  runDestroy(view);
-  view = EmberView.create({
-    items: emberA(['a', 'b', 'c']),
-    template: compile('{{#each view.items as |item|}}{{item}}{{/each}}')
-  });
-
-  runAppend(view);
-
-  run(function() {
-    view.get('items').pushObject('a');
-  });
-
-  equal(view.$().text(), 'abca');
-});
-
-QUnit.test('pushing primitive item twice will render properly', function() {
-  runDestroy(view);
-  view = EmberView.create({
-    items: emberA(),
-    template: compile('{{#each view.items as |item|}}{{item}}{{/each}}')
-  });
-
-  runAppend(view);
-
-  run(function() {
-    view.get('items').pushObject('a');
-  });
-
-  equal(view.$().text(), 'a');
-
-  run(function() {
-    view.get('items').pushObject('a');
-  });
-
-  equal(view.$().text(), 'aa');
-});
-
-
-QUnit.test('duplicate keys work properly with objects', function() {
-  runDestroy(view);
-  let duplicateItem = { display: 'foo' };
-  view = EmberView.create({
-    items: [
-      duplicateItem,
-      duplicateItem,
-      { display: 'bar' },
-      { display: 'qux' }
-    ],
-    template: compile('{{#each view.items as |item|}}{{item.display}}{{/each}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'foofoobarqux');
-});
-
-QUnit.test('pushing a new duplicate key will render properly with objects', function() {
```

Comment:
[Migrated](https://github.com/emberjs/ember.js/pull/13349/commits/c08e75f3474e8bb995f5f24b837479bf3fca64f8#diff-d3c97c178f7e01e59143b5bf2f84062aR242) with the previous test into a single INUR